### ISSUE TITLE
Javax validation support for groups

### ISSUE THREAD

**Issue by hleb-albau** [victools/jsonschema-generator#7]
Hi, it would be nice, to generate schema as follows

```java
ObjectMapper objectMapper = new ObjectMapper();
JavaxValidationModule module = new JavaxValidationModule(group1, group2);
SchemaGeneratorConfigBuilder configBuilder = new SchemaGeneratorConfigBuilder(objectMapper, OptionPreset.PLAIN_JSON)
        .with(module);
SchemaGeneratorConfig config = configBuilder.build();
SchemaGenerator generator = new SchemaGenerator(config);
JsonNode jsonSchema = generator.generateSchema(Yourclass.class);
```

**Comment by CarstenWickner** [comment]
Hi @hleb-albau,

What would these groups be? I cannot tell from the example above.

**Comment by hleb-albau** [comment]
```
public class Person {

@NotEmpty
private String name;

@NotEmpty(groups = ExtenderDescriptionGroup.class)
private String displayName;

}
```

For such class jsr allow you to have two possible validations strategies, depending on group specified during validation.

Than:
```
JavaxValidationModule module = new JavaxValidationModule(ExtenderDescriptionGroup.class);
```
Should return not empty constraint for field `displayName`.
```
JavaxValidationModule module = new JavaxValidationModule();
```
Should return no constrains for field `displayName`.

**Comment by CarstenWickner** [comment]
I like the idea of respecting validation groups.
Thank you for the suggestion and explanation.

I'll look into it.

**Comment by CarstenWickner** [comment]
HI @hleb-albau,

I'm happy to announce that this has been implemented and released under version [v3.2.0](https://github.com/victools/jsonschema-module-javax-validation/releases/tag/v3.2.0) of the `victools/jsonschema-module-javax-validation`.

A new `forValidationGroups(Class<?>...)` method was added on the `JavaxValidationModule`.

The usage would look like this then:
```java
ObjectMapper objectMapper = new ObjectMapper();
JavaxValidationModule module = new JavaxValidationModule()
    .forValidationGroups(ExtenderDescriptionGroup.class);
SchemaGeneratorConfigBuilder configBuilder = new SchemaGeneratorConfigBuilder(objectMapper, OptionPreset.PLAIN_JSON)
    .with(module);
SchemaGeneratorConfig config = configBuilder.build();
SchemaGenerator generator = new SchemaGenerator(config);
JsonNode jsonSchema = generator.generateSchema(YourClass.class);

System.out.println(jsonSchema.toString());
```
